_docs/_latest/configure/explain/index.md

```diff
@@ -1,135 +1,59 @@
-`forseti_iam explainer --help`. You can also explore the `forseti_iam` tool by
-using `forseti_iam --help`.
+Before you start using Explain, you'll first select the data model you
+want to use.
 
-Because Explain and its API are still in early development, third party
-clients won't be supported until a first stable API version is released.
+To review the hierarchy of commands and explore Explain functionality, use
+`–help`.
 
-## Deploying Explain 
+### Selecting a data model
 
-You can use the provided Explain deployment script
-the deployment. Make sure you have the permission inside your organization to
-create projects, set IAM policies, create service accounts and enable the required
-APIs.
-
-You can directly clone the experimental branch into your cloud shell and execute
-the deployment script:
-
-  ```bash
-  $ git clone -b explain-experimental \
-    https://github.com/GoogleCloudPlatform/forseti-security.git
-  $ bash forseti-security/scripts/iam_auto_deployment.sh
-  ```
-
-
-## First start
-
-After the deployment script is finished, allow a few minutes for the virtual machine
-startup script to install all dependencies and start the services. You can observe
-the progress by looking at the deployment log at /tmp/deployment.log.
-## Running the client
-
-The Explain client uses hierarchical command parsing. At the top level,
-commands divide into "explainer" and "playground".
+```bash
+$ forseti model use <YOUR_MODEL_NAME>
+```
 
-### Setting up an explain model
+### Accessing a data model
 
-You can use '--help' to go through the hierarchy of commands and explore
-the functionality. Only the basic examples are shown here.
+Following are some example commands to list and filter a data model.
 
-To run the client, you'll first create an inventory. This process can take
-some time depending on the number of resources in your environment.
+#### Listing all resources in the data model
 
 ```bash
-$ forseti_iam inventory create
-$ forseti_iam inventory list
+$ forseti explainer list_resources
 ```
 
-After the creation of an inventory, create a queryable data model. You need
-to refer to the inventory id as it shows up under the previous list command
+#### Filter the results and list resources only in a folder
 
 ```bash
-$ forseti_iam explainer create_model --id 1 inventory
+$ forseti-client-XXXX-vm> forseti explainer list_resources --prefix organization/1234567890/folder/folder-name
 ```
 
-Once you have a queryable data model, you can switch into it by getting the
-handle of the model and setting an environment variable (IAM_MODEL) or directly
-specifying it in the command line using '--use-model'
+#### List all members in the data model
 
 ```bash
-$ forseti_iam --use-model ... explainer denormalize
+$ forseti-client-XXXX-vm> forseti explainer list_members
 ```
 
-## Running the server
-
-The Explain server uses its own database for IAM models, simulations and inventory.
-
+#### Filter the results and list members with a prefix match
```

Comment:
This content below is *using* forseti explain. This document should be describing the steps and options to configure explain prior to using it.

Reply:
Updated.